packages/lodestar/test/unit/network/reqResp.test.ts

```diff
@@ -217,9 +217,6 @@ describe("[network] rpc", () => {
     const libP2pMock = await createNode(multiaddr);
     libP2pMock.dialProtocol = async (_, __, {signal}: {signal: AbortSignal} ) => {
       timer.tick(TTFB_TIMEOUT);
-      if(signal.aborted) {
```

Comment:
since we throw error ourself in `dialProtocol` when timeout, we don't need this code and the test `dialProtocol - timeout` still passes